utilities/generate_test_locales/pdm.lock

```diff
@@ -0,0 +1,747 @@
+# This file is @generated by PDM.
+# It is not intended for manual editing.
+
+[metadata]
+groups = ["default"]
+strategy = ["cross_platform", "inherit_metadata"]
+lock_version = "4.4.2"
+content_hash = "sha256:61d127fa65b3a44435b5a92c6f4bd8e88d3b9b9701d0f435b0f06974da1c0022"
```

Comment:
I get this change after running the commands.

```suggestion
content_hash = "sha256:a2b1f29fe398ed9f3efec5cbdd7a4e92f59b0a0d73a125d76f7aeb3e45ac9e89"
```

Reply:
Fixed in [a69287d](https://github.com/WordPress/openverse/pull/4626/commits/a69287d276838550da77988e51e0ab31cc524704)